keanu-python/keanu/vertex/iff.py

```diff
@@ -0,0 +1,39 @@
+from typing import Union
+
+from keanu.infer_type import get_type_of_value
+from .base import Boolean
+from .vertex_casting import do_inferred_vertex_cast
+from .base import Vertex, vertex_constructor_param_types
+from .generated import BooleanIf, DoubleIf, IntegerIf, ConstantBoolean, ConstantInteger, ConstantDouble
+
+
+def __cast_to_vertex(input: vertex_constructor_param_types) -> Vertex:
```

Comment:
@christophernorth - this is duplicated from `generated.py`. Before moving it somewhere else I wanted to check with you about the work you did to simplify the dependency tree.

Reply:
Let's chat - but if you depend on .generated - you should be able to just use the version from there rather than copying it.